_posts/clairewlj/2016-04-05-LingjiesMOARdictionary.md

```diff
@@ -5,7 +5,10 @@ title: "Lingjie's MOAR Dictionary Exercise"
 ---
 
 Exercise 1:
-https://trinket.io/python3/d81c5eaf19
+<iframe src="https://trinket.io/embed/python3/d81c5eaf19" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
 
 Exercise 2-5:
-https://trinket.io/python3/178e808bc0
+<iframe src="https://trinket.io/embed/python3/178e808bc0" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Reflection: I really find these exercises helpful to improve my skill of managing data, especially after the experience of participating in the ASA Datafest competition. I found R not that efficient to handle big data over 1 GB size, and I hope I'll do similar work faster wth the help of Python.
```

Comment:
Good thought.  As you look for more efficiency, try to learn the Computer Science comcepts of computational or algorithmic efficiency.  This will give you tools to refactor your code to achieve the same results in less time, regardless of language.  Big O notation is a key component of this that you can search for and learn more about.